ci/sync/main.sh

```diff
@@ -139,7 +142,7 @@ main() {
     while [ "$BLOCK" -lt "$STOP_BLOCK" ]; do
         if [ "$ATTEMPTS" -gt "$MAX_ATTEMPTS" ]; then
             if [ "$NODE_RESTARTS" -lt "$MAX_NODE_RESTARTS" ]; then
-                echo "Node Stuck After $ATTEMPTS attempts, restarting node"
+                echo "Node Stuck After ${ATTEMPTS} attempts, restarting node"
                 $DEFI_CLI_CMD stop
                 sleep 20
```

Comment:
Perhaps, abstract into `stop_node_and_wait` to make things more consistent and deterministic.

Reply:
Resolved with stop_node function in commit a996660